cosmwasm/packages/sgx-vm/src/wasmi/exports.rs

```diff
@@ -28,71 +27,127 @@ pub unsafe fn recover_buffer(ptr: UserSpaceBuffer) -> Option<Vec<u8>> {
 
 /// Read a key from the contracts key-value store.
 #[no_mangle]
-pub extern "C" fn ocall_read_db(context: Ctx, key: *const u8, key_len: usize) -> EnclaveBuffer {
+pub extern "C" fn ocall_read_db(
+    context: Ctx,
+    vm_error: *mut UntrustedVmError,
```

Comment:
Can't we just call this field `error`?

Reply:
it's specific for instances of the `VmError` type. (I'll explain what it is in a bigger comment below)